src/components/admin/HearingFormStep4.js

```diff
@@ -92,12 +93,14 @@ class HearingFormStep4 extends React.Component {
           </Col>
         </Row>
 
-        <TextArea
+        <MultiLanguageTextField
```

Comment:
The closureInfo section is displayed here.

Therefore, closureInfo should not be shown at all in Form step 2 under hearing sections.